src/entity/DisallowHandle.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, EntityManager, Index, PrimaryColumn } from 'typeorm';
+
+@Entity()
+export class DisallowHandle {
+  @PrimaryColumn({ unique: true })
+  @Index({ unique: true })
+  value: string;
+}
+
+export const checkDissalowHandle = async (
+  entityManager: EntityManager,
+  value: string,
+): Promise<boolean> => {
+  const handle = await entityManager
+    .getRepository(DisallowHandle)
+    .createQueryBuilder()
+    .select('value')
+    .where('value = :value', {
+      value,
+    })
+    .getRawOne();
+  return !!handle;
```

Comment:
Don't know if we can cache this since value will almost never change as table would be static.

Reply:
Should be relatively fast. So I wouldn't worry about cache at this point but great thinking

Reply:
Agreed with Ido, great point but for now it's ok, not super heavy.
Also we might be adding new values in the first period.

Reply:
Works 👍